docs/datasource_api_userguide.md

```diff
@@ -222,6 +222,23 @@ The following is TiDB-specific options, which can be passed in through `TiDBOpti
 | writeConcurrency            | -             | false    | maximum number of threads writing data to tikv, suggest `writeConcurrency` <= 8 * `number of TiKV instance` | 0       |
 TiSpark's common options can also be passed in, e.g. `spark.tispark.plan.allow_agg_pushdown`, `spark.tispark.plan.allow_index_read`, etc.
 
+## TiDB Version and Configuration for Write
+TiDB's version should >= 3.0 and make sure that the following config is correctly set.
+
+```
+# enable-table-lock is used to control table lock feature. Default is false, indicate the table lock feature is disabled.
```

Comment:
what do you mean by saying "table lock feature"?

Reply:
copy from tidb https://github.com/pingcap/tidb/blob/master/config/config.toml.example#L57

Reply:
em, I guess they did not pay attention to the comments.

Reply:
```suggestion
# enable-table-lock is used to control table lock feature. The default value is false, indicating the table lock feature is disabled.
```

Reply:
fixed